cmp/ag-grid/AgGrid.scss

```diff
@@ -214,12 +214,14 @@
     // this will cause horizontal mis-alignment between expandable and non-expandable sibling rows
     // (and in general tighter padding preferred here vs. full-width groups).
     .ag-row {
-      .group-header-icon-expanded {
+      .ag-group-expanded {
         margin: 0 var(--xh-pad-half-px) 0 0;
+        width: 10px;
```

Comment:
The comment mentions removing a margin, but then we are putting in a hard-coded pixel width here.  We will have to review, but on the face of it I don't understand what we're looking to achieve here.  The comment and the code change don't seem to match.

In general we are also looking to avoid the hard-coded use of specific sizes wherever possible, in favor of our standard CSS vars for padding.

Reply:
The 'expanded' and 'contracted' icons are different sizes. This means that expanding or contracting a row will cause the row to move around. The comment was not updated to reflect this.

I certainly agree with avoiding hard-coded magic numbers.